qemu/tests/image_locking_read_test.py

```diff
@@ -0,0 +1,75 @@
+import logging
+
+from avocado import fail_on
+
+from virttest import env_process
+from virttest import error_context
+from virttest import virt_vm
+
+from qemu.tests.qemu_disk_img import QemuImgTest
+
+
+@error_context.context_aware
+def run(test, params, env):
+    """
+    Verification that image lock has no effect on the read operation from
+    different image chain.
+    Steps:
+        1. create the first snapshot chain: image1 -> sn01 -> sn02
+        2. boot first vm from sn02
+        3. create the second snapshot chain: image1 -> sn11 -> sn12 ->sn13
+        4. boot second vm frm sn13 and create a temporary file
+        5. commit sn13
```

Comment:
need add check whether the snapshot file emptied,please refer to 1719107

Reply:
There is no such a step in the case and this scenario is covered by other cases.